examples/multiphysics/static-bubble-chns/outputs/R_0.15/static-bubble-chns.prm

```diff
@@ -0,0 +1,164 @@
+# Listing of Parameters
```

Comment:
Since there is a generator for these various cases, maybe you don't need them in the example directory and can let users generate them themselves

Reply:
Yes, I will only leave the generating and launching scripts